.NET/Microsoft.Recognizers.Definitions.Common/Microsoft.Recognizers.Definitions.Common.csproj

```diff
@@ -379,12 +391,27 @@
     </None>
     <None Update="Spanish\NumbersWithUnitDefinitions.tt">
     </None>
+    <None Update="Swedish\ChoiceDefinitions.tt">
+      <Generator>TextTemplatingFileGenerator</Generator>
+    </None>
+    <None Update="Swedish\NumbersDefinitions.tt">
+      <Generator>TextTemplatingFileGenerator</Generator>
+      <CustomToolNamespace></CustomToolNamespace>
```

Comment:
We should be careful to follow exactly the same attributes are the structure for other cultures. For some reason it seems the new entries are a little inconsistent.

Reply:
Fixed